kafka-client/src/main/java/dev/responsive/kafka/clients/EndOffsetsPoller.java

```diff
@@ -217,21 +231,22 @@ public void onPartitionsAssigned(final Collection<TopicPartition> assigned) {
       }
     }
 
-    private void update(final Map<TopicPartition, ListOffsetsResultInfo> endOffsets) {
-      for (final var entry : endOffsets.entrySet()) {
-        this.endOffsets.computeIfPresent(entry.getKey(), (k, v) -> {
-          logger.info("update end offset of {}/{} to {}",
-              k.topic(),
-              k.partition(),
-              entry.getValue().offset()
-          );
-          return entry.getValue().offset();
-        });
+    private void update(final Map<TopicPartition, ListOffsetsResultInfo> newEndOffsets) {
+      for (final var entry : newEndOffsets.entrySet()) {
+        endOffsets.computeIfPresent(entry.getKey(), (k, v) -> entry.getValue().offset());
+      }
+
+      if (log.isDebugEnabled()) {
+        log.debug("Updated end offsets to [{}].",
```

Comment:
DEBUG feels more appropriate to me, but I could definitely be persuaded to bump this up to INFO if it seems useful -- once every 30s isn't too bad for a single log line, but it could get pretty long for large clusters with a lot of partitions. Anyways I'm fine either way